Menu.cs

```diff
@@ -22,7 +22,7 @@ public string Answer()
 
         public void Print()
         {
-            Console.WriteLine(Title);
+            Console.WriteLine("\n{0}\n-----------------------------",Title);
```

Comment:
This was just a cosmetic thing to make the menu look more like a menu the format turns into:
```

<TITLE>
---------------------
```